osu.Game/OsuGameBase.cs

```diff
@@ -206,7 +206,7 @@ public virtual string Version
         /// </summary>
         private readonly FramedBeatmapClock beatmapClock = new FramedBeatmapClock(true);
 
-        protected override Container<Drawable> Content => content;
+        protected override Container<Drawable> Content => content ?? base.Content;
```

Comment:
This looks sketchy as all hell, was this only to change that one `base.Content.Add` call? Super not worth it if you ask me. Is this actually necessary for any of this to work?

Reply:
More like replace requiring to use `base.Content.Add` everywhere with `Add`. If you insist, I can bring it back to use the former.

Reply:
I think I prefer that to "`Add()` suddenly changes semantics midway through BDL". But maybe that's just me. Dunno.

Reply:
How many calls would need to be changed to `base.Content.Add`? I agree it's weird having it start adding to a different location *at some point in time*. We've done this elsewhere, but it's usually in tests where it doesn't matter. I don't know how much it makes sense at the game level.

It also makes it very hard to see what the actual fix was in this diff. If anything I'd prefer it to be done separately as a code quality change, leaving the `AddInternal` calls where they can still exist amicably for now.

Reply:
I could change to `base.Content.Add`, but not `AddInternal` since that missed the whole point of this PR.

Reply:
Let's start with that to see how things look.